### PR TITLE
Base url warning

### PR DESCRIPTION
**Pull request by Artiume**
Everything breaking

### CODE REVIEW

File: general/administration/connectivity.md [hunk 1]
```diff
@@ -59,6 +59,10 @@ Live TV devices will often use a random UDP port for HD Homerun devices. The ser
 
 ## Base URL
 
+> [!WARNING]
+
+> Base URL is known to break HD Homerun, DLNA, Sonarr, Radarr, Chromecast, and MrMC.
```
Comment (dkanada) [inline general/administration/connectivity.md]:
```suggestion
> Base URL is known to break HD HomeRun, DLNA, Sonarr, Radarr, Chromecast, and MrMC.
```

File: general/administration/connectivity.md [hunk 2]
```diff
@@ -59,6 +59,10 @@ Live TV devices will often use a random UDP port for HD Homerun devices. The ser
 
 ## Base URL
 
+> [!WARNING]
+
```
Comment (dkanada) [inline general/administration/connectivity.md]: This newline might break the note formatting.

**Comment by PrplHaz4** [comment]
I thought at least the DLNA part of this was fixed here: https://github.com/jellyfin/jellyfin/pull/2054

**Comment by Artiume** [comment]
It's a general warning for when if you are having issues, people can try it first before coming to us.

**Pull request merged**